app/models/suggestion.py

```diff
@@ -1,22 +1,27 @@
 from datetime import datetime
 import pytz
+
 from .. import db
+from ..models.resource import Resource, ResourceBase
+
 
-class Suggestion(db.Model):
+class ResourceSuggestion(ResourceBase):
     """
     Association between a resource and potential suggestions for it
     """
-    __tablename__ = 'suggestions'
-    id = db.Column(db.Integer, primary_key=True)
-    resource_id = db.Column(db.Integer, db.ForeignKey('resources.id'))
-    suggestion_text = db.Column(db.String(250))
```

Comment:
please change references to old name here: https://github.com/hack4impact/asylum-connect-catalog/blob/veronica-resources/tests/test_suggestions_model.py